pkg/db/src/implementations/version-1.2/books.ts

```diff
@@ -37,10 +37,29 @@ class Books implements BooksInterface {
 	}
 
 	async get(isbns: string[]): Promise<(BookEntry | undefined)[]> {
-		const rawBooks = await this.#db._pouch.allDocs<BookEntry>({ keys: isbns.map((isbn) => `books/${isbn}`), include_docs: true });
+		const rawBooks = await this.#db._pouch
```

Comment:
These aren't `rawBooks` anymore as the documents get unwrapped, might as well name the var `books`